packages/@ourworldindata/grapher/src/core/GrapherConstants.ts

```diff
@@ -103,15 +103,15 @@ export enum ScatterPointLabelStrategy {
 
 export type SeriesName = string
 
-export const GRAPHER_VARIABLES_ROUTE = "/grapher/data/variables"
-export const GRAPHER_VARIABLE_DATA_ROUTE = `${GRAPHER_VARIABLES_ROUTE}/data`
-export const GRAPHER_VARIABLE_METADATA_ROUTE = `${GRAPHER_VARIABLES_ROUTE}/metadata`
-
-export const getVariableDataRoute = (variableId: number): string =>
-    `${GRAPHER_VARIABLE_DATA_ROUTE}/${variableId}.json`
-
-export const getVariableMetadataRoute = (variableId: number): string =>
-    `${GRAPHER_VARIABLE_METADATA_ROUTE}/${variableId}.json`
+export const getVariableDataRoute = (
+    dataBaseUrl: string,
+    variableId: number
+): string => `${dataBaseUrl}data/${variableId}.json`
+
+export const getVariableMetadataRoute = (
+    dataBaseUrl: string,
+    variableId: number
+): string => `${dataBaseUrl}metadata/${variableId}.json`
```

Comment:
In [the other PR](https://github.com/owid/owid-grapher/pull/1939#issuecomment-1503770026) Lars wrote that we should go with `123.data.json` and `123.metadata.json`.